shared/src/main/scala/doodle/core/Angle.scala

```diff
@@ -3,7 +3,7 @@ package doodle.core
 import scala.annotation.tailrec
 
 /**
-  * An angle in radians, normalized to be in [0, 2pi)
+  * An angle in radians, normalized to be in [0, 2pi]
```

Comment:
Is this change correct? Is it [open or closed](http://en.wikipedia.org/wiki/Interval_%28mathematics%29#Terminology) at the upper bound?